rest-twitter/service/application.py

```diff
@@ -0,0 +1,104 @@
+from flask import Flask, request, jsonify
+import pymongo
+from bson.json_util import dumps
+from datetime import datetime 
+import json 
+import computeSent
+
+# Connect to Mongo Store
+client = pymongo.MongoClient("mongodb://jpvm:jpvmtdb@18.191.169.163/tweets_db")
+
+
+# Use tweets database
+db = client.tweets_db 
+
+# Use sentiment collection
+sentiments = db.sentiment_collection 
+
+
+# Elastic Beanstalk app setup
+
+# EB looks for an 'application' callable by default
+application = Flask(__name__)
+
+# function to get current timestamp used for get endpoint query parameters
+def get_timestamp():
+    return datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
```

Comment:
can we put it in config

Reply:
I left this one in the application file since we have to tag each analyzed tweet with it.